cloudsmith/data_source_package_download_test.go

```diff
@@ -0,0 +1,47 @@
+package cloudsmith
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/cloudsmith-io/cloudsmith-api-go"
+)
+
+func TestDownloadPackage(t *testing.T) {
```

Comment:
This is really only manually testing a couple of the functions that have been added in the PR, could this be modified to be a real terraform integration test? The other resources and data sources should have examples you can copy.